### PR TITLE
Add confguration->configuration

### PR DESCRIPTION
**Pull request by waldyrious**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -6402,6 +6402,7 @@ confety->confetti
 conffiguration->configuration
 confgiuration->configuration
 confgiured->configured
+confguration->configuration
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:6405]: Do you want to add -e and -ed variants of this too please?

**Review by peternewman — CHANGES_REQUESTED** [review]
LGTM

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (waldyrious) [reply codespell_lib/data/dictionary.txt:6405]: Sure.

Comment (waldyrious) [reply codespell_lib/data/dictionary.txt:6406]: @peternewman I took the liberty to also add `confgiure->configure`, for completeness.

Comment (peternewman) [reply codespell_lib/data/dictionary.txt:6406]: I think that's what I meant by -e...

Comment (waldyrious) [reply codespell_lib/data/dictionary.txt:6406]: I was referring to the existing `confgiur-` stem (different from this PR's original `confgur-` one), which already had the `-ure` and `-ed` endings, but not `-e`.

Comment (peternewman) [reply codespell_lib/data/dictionary.txt:6406]: Yeah, I think I realised that eventually...

**Review by peternewman — APPROVED** [review]
LGTM thank

**Pull request merged**